Assignment 3.Rmd

```diff
@@ -26,7 +29,7 @@ The algorithm will treat each row as a value belonging to a person, so we need t
 
 ```{r}
 
-K2 <- 
+K2 <- K1[-(1)]
 
 ```
```

Comment:
It looks like you have deleted the question about the missing data. What do you think deleting the missing students means for our analysis?